### PR TITLE
chore: rework project structure

### PR DESCRIPTION
**Pull request by JeremyPansier**
# Checklist:
- [ ] The code follows our code [conventions](https://github.com/my-cloud/ruthenium/blob/main/.github/CONTRIBUTING.md#go).
- [ ] Important principle changes have been documented in the [wiki](https://github.com/my-cloud/ruthenium/wiki).
- [ ] The new code is covered by unit tests with assertions.
- [ ] The behaviour [node](https://github.com/my-cloud/ruthenium/blob/main/src/node) and [UI](https://github.com/my-cloud/ruthenium/blob/main/src/ui) have been manually tested and is as expected.
- [ ] The squash commit message follows our [conventions](https://github.com/my-cloud/ruthenium/blob/main/.github/CONTRIBUTING.md#git).

### CODE REVIEW

File: deploy/helm/templates/statefulset.yaml [hunk 1]
```diff
@@ -66,6 +66,8 @@ spec:
             memory: {{ $container.resources.requests.memory }}
           limits:
             memory: {{ $container.resources.limits.memory }}
+            cpu: 0.5
```
Comment (gwenall) [inline deploy/helm/templates/statefulset.yaml:69]: The simplest way is to add the cpu and storage limits in the values.yaml file.
I believe the linter will accept that.
```
    resources:
      requests:
        memory: "128Mi"
        cpu: 0.5
        storage: 2GI
      limits:
        memory: "512Mi"
        cpu: 2
        storage: 4Gi
```


worst case,  if the above doesn't work as expected, you can bury the default values in this STS manifest with something like
`cpu:  {{ $container.resources.limits.cpu }} | default 0.5`

**Review by gwenall — CHANGES_REQUESTED** [review]

File: deploy/helm/templates/statefulset.yaml (hunk 1 above)
Comment (JeremyPansier) [reply deploy/helm/templates/statefulset.yaml]: done

File: deploy/helm/values.yaml [hunk 2]
```diff
@@ -79,8 +83,12 @@ app:
     resources:
       requests:
         memory: "128Mi"
+        cpu: 0.5
+        storage: 2GI
```
Comment (gwenall) [inline deploy/helm/values.yaml:87]: Gi  not GI

**Review by gwenall — CHANGES_REQUESTED** [review]

File: deploy/helm/values.yaml (hunk 2 above)
Comment (JeremyPansier) [reply deploy/helm/values.yaml:87]: Thx !

Comment (JeremyPansier) [reply deploy/helm/values.yaml]: done

**Review by gwenall — APPROVED** [review]

File: deploy/helm/values.yaml (hunk 2 above)
Comment (gwenall) [reply deploy/helm/values.yaml]: all good

**Pull request merged**